### PR TITLE
DDLS: Write description in sentence style

### LINKED ISSUE #441: Descriptions are title style for DDLS
Descriptions in ABAP file format of DDLS are title style. They shall be sentence style.

### PR DESCRIPTION
**Pull request by BeckerWdf**
Fixes #441

### CODE REVIEW

**Comment by BeckerWdf** [comment]
I think "ABAP Development Tools" is a name and should stay "ABAP Development Tools" even as description.

**Comment by schneidermic0** [comment]
> I think "ABAP Development Tools" is a name and should stay "ABAP Development Tools" even as description.

I have just looked it up in SAPterm. You are right.

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Thanks, @BeckerWdf. I guess you want to update "ABAP Development Tools"

I found some more locations. Do update these, too?

- https://github.com/SAP/abap-file-formats/blob/34d18084c75af98a2b60b413039ccd0b66499584/file-formats/ddls/type/zif_aff_ddls_v1.intf.abap#L4-L12
- https://github.com/SAP/abap-file-formats/blob/34d18084c75af98a2b60b413039ccd0b66499584/file-formats/ddls/type/zif_aff_ddls_v1.intf.abap#L95-L105

**Comment by BeckerWdf** [comment]
> I found some more locations. Do update these, too?

Yes I will.

File: file-formats/ddls/type/zif_aff_ddls_v1.intf.abap [hunk 1]
```diff
@@ -16,69 +16,69 @@ INTERFACE zif_aff_ddls_v1
...
+      "! Projection view
       projection_view    TYPE ty_source_type VALUE 'P',
     END OF co_source_type,
 
     "! <p class="shorttext">Source Origin</p>
     "! Source origin
     BEGIN OF co_source_origin,
       "! <p class="shorttext">ABAP Development Tools</p>
-      "! ABAP Development Tools
+      "! ABAP development tools
```
[the remark this answers is not in the document]
Comment (schneidermic0) [reply file-formats/ddls/type/zif_aff_ddls_v1.intf.abap:54]: I think this is still open and should be reverted, shouldn't it?

[the remark this answers is not in the document]
Comment (BeckerWdf) [reply file-formats/ddls/type/zif_aff_ddls_v1.intf.abap:54]: Yes

**Review by schneidermic0 — APPROVED** [review]
Looks good to me. Thanks :)

**Pull request merged**